### ISSUE TITLE
Hook Error: Expected 1 JSON task(s), found 0

### ISSUE THREAD

**Issue by languitar** [tbabej/taskwarrior-block-recurrence-hook#1]
I am trying to use this hook in task 2.5.1 but on `task next` I receive this error message:

```
Hook Error: Expected 1 JSON task(s), found 0
```

**Comment by tbabej** [comment]
Indeed, this hook does not work anymore. This should be probably noted in the README.

It has been deprecated since TW native solution is now available. Simply use recurrence=no setting in your .taskrc to disable recurrence.

**Comment by languitar** [comment]
Ok, thanks. Adding this to the readme would be nice.
